server/routes/visitJourney/checkYourBooking.test.ts

```diff
@@ -184,12 +185,23 @@ testJourneys.forEach(journey => {
           .expect(302)
           .expect('location', `${journey.urlPrefix}/confirmation`)
           .expect(() => {
-            expect(visitService.bookVisit).toHaveBeenCalledWith({
-              username: 'user1',
-              applicationReference: visitSessionData.applicationReference,
-              applicationMethod: visitSessionData.requestMethod,
-              allowOverBooking: false,
-            })
+            if (journey.isUpdate) {
```

Comment:
Hmm, I think it would be better as a separate test as you're adding logic inside the test itself, even if it's a complete duplicate of this test. Otherwise we don't know what this test is doing just looking at the results. Not sure what @tpmcgowan thinks.

Reply:
Will address this now, I'll admit it didn't feel correct at the time. Will pull this logic out into seperate test